### PR TITLE
Rename update output port

### PR DESCRIPTION
**Pull request by YohannParis**
This pull request includes several changes focused on standardizing the event names in the `tera` components and improving code clarity. The changes primarily involve renaming the `update-output-port` event to `update-output` across various components and updating related function names accordingly.

Event name standardization:

* [`tera-code-asset-wrapper.vue`](diffhunk://#diff-87c6953826aa0eda7464920fac516185b0b9b8360e86d3053c702ce7a070f7b7L28-R35): Renamed the event `update-output-port` to `update-output` and updated the corresponding emit call in the `onApplyChanges` function.
* [`tera-intervention-policy-drilldown.vue`](diffhunk://#diff-54c31526a779022f9231558c720e27ce9e02ed6da5df3ff490e94bd8bc891248L208-R208): Removed the `update-output-port` event from the `defineEmits` call.
* [`tera-model-config-drilldown.vue`](diffhunk://#diff-9c14e9c321bc9b9b31d04842da2fdcbe723be5b0cbb908af74463564add63cceL266-R266): Removed the `update-output-port` event from the `defineEmits` call.
* [`ptera-model-edit.vue`](diffhunk://#diff-0a13f09762f1288ac204699953f96297a5753d282dd381ca4e4bc1bbb589af7fL135-R135): Renamed the event `update-output-port` to `update-output` and updated the corresponding emit call in the `updateNode` function. [[1]](diffhunk://#diff-0a13f09762f1288ac204699953f96297a5753d282dd381ca4e4bc1bbb589af7fL135-R135) [[2]](diffhunk://#diff-0a13f09762f1288ac204699953f96297a5753d282dd381ca4e4bc1bbb589af7fL407-R407)
* [`era-model-from-equations-drilldown.vue`](diffhunk://#diff-2f796b404e2219b0045f6ee5141b6caa46be4372ebd0720307afca92e0b412f2L186-R186): Renamed the event `update-output-port` to `update-output` and updated the corresponding emit call in the `updateNodeLabel` function. [[1]](diffhunk://#diff-2f796b404e2219b0045f6ee5141b6caa46be4372ebd0720307afca92e0b412f2L186-R186) [[2]](diffhunk://#diff-2f796b404e2219b0045f6ee5141b6caa46be4372ebd0720307afca92e0b412f2L380-R380)
* [`tera-stratify-mira.vue`](diffhunk://#diff-a458f50e7b6b63c930542f560b14920df8e46d18f5c02237dee5ab394fbcdc94L155-R155): Renamed the event `update-output-port` to `update-output` and updated the corresponding emit call in the `updateNodeOutput` function. [[1]](diffhunk://#diff-a458f50e7b6b63c930542f560b14920df8e46d18f5c02237dee5ab394fbcdc94L155-R155) [[2]](diffhunk://#diff-a458f50e7b6b63c930542f560b14920df8e46d18f5c02237dee5ab394fbcdc94L469-R469)

Code clarity improvements:

* [`tera-workflow.vue`](diffhunk://#diff-3ce4d75a5f0d2a6e1ece0fde9f65c23749895a1e2143a101bde090f7a908be76R156-L166): Updated the component bindings and event handlers to use `update-output` instead of `update-output-port`. [[1]](diffhunk://#diff-3ce4d75a5f0d2a6e1ece0fde9f65c23749895a1e2143a101bde090f7a908be76R156-L166) [[2]](diffhunk://#diff-3ce4d75a5f0d2a6e1ece0fde9f65c23749895a1e2143a101bde090f7a908be76L412-R414)
* [`services/workflow.ts`](diffhunk://#diff-b982e1f4bbcd8bca4c8812281c0bdb0b90cca4937e0b83547f3053daf02f0e74L810-R819): Updated the function `updateOutputPort` to `updateOutput` and modified the implementation to use the new event name.

### CODE REVIEW

File: packages/client/hmi-client/src/services/workflow.ts [hunk 1]
```diff
@@ -807,10 +807,16 @@ export function getActiveOutput(node: WorkflowNode<any>) {
...
+/**
+ * Update the output of a node referenced by the output id
+ * @param node
+ * @param updatedOutput
+ */
+export function updateOutput(node: WorkflowNode<any>, updatedOutput: WorkflowOutput<any>) {
+	const foundOutput = node.outputs.find((output) => output.id === updatedOutput.id);
+	if (foundOutput) {
+		Object.assign(foundOutput, updatedOutput);
+	}
```
Comment (mwdchang) [inline packages/client/hmi-client/src/services/workflow.ts:819]: So frankly I don't know why we want to go through the exercise of renaming this - as it had been mentioned several times before, this is an escape hatch, and there is no need nor desire to make it seem like it is a legitimate api-interface.

But if you really want to do this, please also update the workflow specification to reflect this change
https://docs.google.com/document/d/1mHEeViL5TpA90p2XtheWvkKR8ofcvieDdi4wWIIkbv8/edit

Comment (YohannParis) [reply packages/client/hmi-client/src/services/workflow.ts:819]: Done

File: packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue [hunk 2]
```diff
@@ -377,7 +377,7 @@ function updateNodeLabel(id: string, label: string) {
 	const outputPort = cloneDeep(props.node.outputs?.find((port) => port.id === id));
 	if (!outputPort) return;
 	outputPort.label = label;
-	emit('update-output-port', outputPort);
+	emit('update-output', outputPort);
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue:380]: I think @blanchco made the labels computed properties, so this function `updateNodeLabel` may not be needed anymore.

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue [hunk 3]
```diff
@@ -466,7 +466,7 @@ function updateNodeOutput(model: Model) {
 	if (!outputPort) return;
 	outputPort.label = model.header.name;
 
-	emit('update-output-port', outputPort);
+	emit('update-output', outputPort);
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:469]: Ditto, see https://github.com/DARPA-ASKEM/terarium/pull/5000/files#r1782785989

File: packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue (hunk 2 above)
Comment (YohannParis) [reply packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue]: Thank you for the catch, talked with @blanchco to confirm what I was doing.

**Review by mwdchang — APPROVED** [review]
I didn't test every operator changed here, looks okay!

**Pull request merged**